Extension/src/LanguageServer/Providers/codeActionProvider.ts

```diff
@@ -92,13 +92,18 @@ export class CodeActionProvider implements vscode.CodeActionProvider {
             let title: string = getLocalizedString(command.localizeStringParams);
             let wsEdit: vscode.WorkspaceEdit | undefined;
             let codeActionKind: vscode.CodeActionKind = vscode.CodeActionKind.QuickFix;
-            if (command.edit) {
+            if (command.edit && command.command !== "C_Cpp.AddMissingInclude") {
```

Comment:
This should be
```
if (command.edit) {
    wsEdit = new vscode.WorkspaceEdit();
    wsEdit.replace(document.uri, makeVscodeRange(command.edit.range), command.edit.newText);
    if (command.command !== "C_Cpp.AddMissingInclude") {
        // Inline macro feature.
        codeActionKind = CodeActionProvider.inlineMacroKind;
        hasInlineMacro = true;
    }
} else if ...
```

Reply:
Also my PR at https://github.com/microsoft/vscode-cpptools/pull/11467 changes `edit` to `workspaceEdits` so I think you'll need to make some minor changes after that gets in.